kythe/cxx/indexer/cxx/GraphObserver.h

```diff
@@ -835,6 +835,12 @@ class GraphObserver {
     return std::nullopt;
   }
 
+  /// \brief Records a denotes relationship as an edge in the graph.
+  /// \param DenoterId The identifier for the denoting node.
```

Comment:
Can you add a bit about the concrete vs abstract idea?

And a bit in the schema, now that I think about it?